frontend/package.json

```diff
@@ -57,12 +58,12 @@
     "eject": "react-scripts eject",
     "compile-scss": "node-sass --include-path ./src/scss --include-path ./node_modules/uswds/dist/scss --output-style compressed -o ./src/styles/ ./src/scss/App.scss",
     "watch-css": "yarn compile-scss && node-sass --include-path ./src/scss --include-path ./node_modules/uswds/dist/scss -o src/styles/ --watch --recursive src/scss/App.scss",
-    "deploy-azure-dev": "REACT_APP_OKTA_URL=https://dev.simplereport.gov REACT_APP_APPINSIGHTS_KEY=29a75c50-8485-4d80-8e1b-41b1a091abb6 REACT_APP_BACKEND_URL=https://dev.simplereport.gov/api/graphql PUBLIC_URL=/app/ yarn build && az storage blob upload-batch -s build/ -d '$web' --account-name simplereportdevapp --destination-path '/app'",
-    "deploy-azure-test": "REACT_APP_OKTA_URL=https://test.simplereport.gov REACT_APP_BACKEND_URL=https://test.simplereport.gov/api/graphql PUBLIC_URL=/app/ yarn build && az storage blob upload-batch -s build/ -d '$web' --account-name simplereporttestapp --destination-path '/app'",
-    "deploy-azure-pentest": "REACT_APP_OKTA_URL=https://pentest.simplereport.gov REACT_APP_BACKEND_URL=https://pentest.simplereport.gov/api/graphql PUBLIC_URL=/app/ yarn build && az storage blob upload-batch -s build/ -d '$web' --account-name simplereportpentestapp --destination-path '/app'",
-    "deploy-azure-demo": "REACT_APP_OKTA_URL=https://demo.simplereport.gov REACT_APP_BACKEND_URL=https://demo.simplereport.gov/api/graphql PUBLIC_URL=/app/ yarn build && az storage blob upload-batch -s build/ -d '$web' --account-name simplereportdemoapp --destination-path '/app'",
-    "deploy-azure-stg": "REACT_APP_OKTA_URL=https://stg.simplereport.gov REACT_APP_BACKEND_URL=https://stg.simplereport.gov/api/graphql PUBLIC_URL=/app/ yarn build && az storage blob upload-batch -s build/ -d '$web' --account-name simplereportstgapp --destination-path '/app'",
```

Comment:
My Spring-foo is weak: where did the `/graphql` URL suffix get removed in the backend?

Reply:
`/graphql` was not removed from the backend. This is a change I forgot to clean up. My initial approach was to us the existing upload API but then I found `apollo-upload-client` and realized it was a lot easier and cleaner then adding another API client to the frontend